docs/sources/panels/visualizations/time-series/change-axis-display.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Change axis display"
+keywords = ["grafana", "time series panel", "documentation", "guide", "graph"]
+weight = 400
++++
+
+# Change axis display
+
+> **Note:** This is a beta feature. Time series panel is going to replace the Graph panel in the future releases.
+
+This section explains how to use Time series field options to control the display of axes in the visualization and illustrates what the axis options do.
+
+Use the following field settings to refine how your axes display.
+
+For more information about applying these options, refer to:
+
+- [Configure all fields]({{< relref "../../field-options/configure-all-fields.md" >}})
+- [Configure specific fields]({{< relref "../../field-options/configure-specific-fields.md" >}})
+
+Some field options will not affect the visualization until you click outside of the field option box you are editing or press Enter.
+
+## Placement
+
+Select the placement of the Y-axis.
+
+### Auto
+
+Grafana automatically assigns Y-axis to the series. When there are two or more series with different units, then Grafana assigns the left axis to the first unit and right to the following units.
+
+### Left
+
+Display all Y-axes on the left side.
+
+![Left Y-axis example](/img/docs/time-series-panel/axis-placement-left-7-4.png)
+
+### Right
+
+Display all Y-axes on the right side.
+
+![Right Y-axis example](/img/docs/time-series-panel/axis-placement-right-7-4.png)
+
+### Hidden
+
+Hide the Y-axes.
+
+![Hidden Y-axis example](/img/docs/time-series-panel/axis-placement-hidden-7-4.png)
+
+## Label
+
+Set a Y-axis text label.
+
+![Label example](/img/docs/time-series-panel/label-example-7-4.png)
+
+If you have more than one Y-axis, then you can give assign different labels in the Override tab.
+
+## Width
+
+Set a fixed width of the axis. By default, Grafana dynamically calculates the width of an axis.
```

Comment:
could be useful to mention why someone may want this. i think the main use case is vertical plotting area alignment. (https://github.com/grafana/grafana/issues/4469). to get full alignment capability we still need to add explicit control over padding.

Reply:
Could you expand on that or suggest verbiage? I literally don't know what use cases there are for this.

Reply:
can i expand on #4469? no, i cannot; there's a very clear explanation in that issue already. please read it.

Reply:
How about something like, "By setting the width of the axis, data whose axis types are different can share the same display proportions. This makes it easier to compare more than one graph’s worth of data because the axes are not shifted or stretched within visual context of each other."

Reply:
Thanks, @osg-grafana! Updated.